Topic3/src/org/maugiam/tdd/main/AmountConverter.java

```diff
@@ -0,0 +1,79 @@
+package org.maugiam.tdd.main;
+
+import java.util.HashMap;
+
+public class AmountConverter {
+
+	protected HashMap<Integer, String> numbers;

```

Comment:
Warning: always use the Interface type when the variable is declared. In this case Map.